families/seth/src/sawtooth_seth/processor/handler/state_manager.go

```diff
@@ -165,3 +166,45 @@ func (mgr *StateManager) MustSetEntry(vmAddress *EvmAddr, entry *EvmEntry) {
 		))
 	}
 }
+
+func (mgr *StateManager) GetBlockInfoConfig() (*BlockInfoConfig, error) {
+	// Retrieve block info config from global state
+	entries, err := mgr.state.Get([]string{CONFIG_ADDRESS})
+	if err != nil {
+		return nil, err
+	}
+	entryData, exists := entries[CONFIG_ADDRESS]
+	if !exists {
+		return nil, nil
+	}
+
+	// Deserialize the entry
+	entry := &BlockInfoConfig{}
+	err = proto.Unmarshal(entryData, entry)
+	if err != nil {
+		return nil, err
+	}
+
+	return entry, nil
+}
+
+func (mgr *StateManager) GetBlockInfo(blockInfoAddr string) (*BlockInfo, error) {
```

Comment:
This should probably just take a uint64 as an argument and construct the address.